extension/memorylimiterextension/factory.go

```diff
@@ -0,0 +1,37 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package memorylimiterextension // import "go.opentelemetry.io/collector/extension/memorylimiterextension"
+
+//go:generate mdatagen metadata.yaml
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/extension"
+)
+
+const (
+	// The value of "type" Attribute Key in configuration.
+	typeStr = "memory_limiter"
+)
```

Comment:
Not needed anymore. Use generated `metadata.Type`

Reply:
replaced with using meta